Fix: Blood of the Karui Life recovered and wording

Sinured opened the pull request:
Fixes #3728 .

### Description of the problem being solved:
1. The current version doesn't have the "50% increased Life Recovered" affix.
2. Wording fix from "increased Amount recovered" to "increased Life recovered

### Before screenshot:
1. 
![karui_current_before](https://user-images.githubusercontent.com/100538010/183893890-aba2ef12-24db-44eb-98f2-52ccf477c092.png)
2.
![karui_before](https://user-images.githubusercontent.com/100538010/183893913-dc207ec7-92ab-4cc0-a296-1c93bf68c420.png)

### After screenshot:
1.
![karui_current_after](https://user-images.githubusercontent.com/100538010/183893947-eb7cc803-9bbf-4e76-b084-d9e58c5fede7.png)
2.
![karui_after](https://user-images.githubusercontent.com/100538010/183893987-c224da91-610d-4745-887a-ae01b45c9084.png)

QuickStick123 commented:
This is what the versions should look like https://www.poewiki.net/wiki/Blood_of_the_Karui so you should probably fix these and update the wording of recovery speed as well and fix the other current issues.
Current:
50% increased Life Recovered
(50-35)% reduced Recovery rate
Recover Full Life at the end of the Flask Effect
Pre 3.16.0
100% increased Life Recovered
(5-20)% increased Recovery Rate
Recover Full Life at the end of the Flask Effect
Pre 3.15.0
50% increased Life Recovered
(5-20)% increased Recovery Rate
Recover Full Life at the end of the Flask Effect
Pre 2.6.0
(5-20)% increased Recovery Rate
Recover Full Life at the end of the Flask Effect
Cannot gain Life during effect
Pre 1.3.0
(20-30)% reduced Recovery Rate
Recover Full Life at the end of the Flask Effect
Cannot gain Life during effect

Sinured commented:
fixed wording and added support for Pre 1.3.0 Version. Additionally I also changed the wording from "Recovery Speed" to "Recovery Rate" for Lavianga's Spirit

QuickStick123 commented:
I would probably fix the mod order 
Cannot gain Life during effect -> move to end of mods.
% increased Life Recovered / reduced -> move to front of mods

And fix the wording.
Cannot Gain Life During Flask Effect -> Cannot gain Life during effect
Recover Full Life at the end of the Flask effect -> Recover Full Life at the end of the Flask Effect
Recovery Rate -> Recovery rate

Sinured commented:
> I would probably fix the mod order Cannot gain Life during effect -> move to end of mods. % increased Life Recovered / reduced -> move to front of mods
> 
> And fix the wording. Cannot Gain Life During Flask Effect -> Cannot gain Life during effect Recover Full Life at the end of the Flask effect -> Recover Full Life at the end of the Flask Effect Recovery Rate -> Recovery rate

done

QuickStick123 commented:
LGTM you just missed Effect is capitalised in "Recover Full Life at the end of the Flask effect"

Sinured commented:
Damn. These details are killing me :D

QuickStick123 commented:
Yeah I normally just copy paste all the lines when I update uniques to make sure I don't miss something. 
But good job nice first pull request. :)
